bc_obps/service/report_service.py

```diff
@@ -85,22 +90,26 @@ def get_report_operation_by_version_id(cls, report_version_id: int) -> ReportOpe
     def save_report_operation(cls, report_version_id: int, data: ReportOperationIn) -> ReportOperation:
         # Fetch the existing report operation
         report_operation = ReportOperation.objects.get(report_version__id=report_version_id)
+        report_operation_representatives = ReportOperationRepresentative.objects.filter(
+            report_version=report_version_id
```

Comment:
this will always return an empty list, we're equating a model (ReportVersion type) to an id (int type)